lib/version.lds

```diff
@@ -160,3 +160,69 @@ LIBKCAPI_1.3.0 {
 	kcapi_get_maxsplicesize;
 	kcapi_set_maxsplicesize;
 } LIBKCAPI_1.2.1;
+
+LIBKCAPI_1.3.1 {
+	kcapi_cipher_encrypt;
+	kcapi_cipher_encrypt_aio;
+	kcapi_cipher_decrypt;
+	kcapi_cipher_decrypt_aio;
+	kcapi_cipher_stream_init_enc;
+	kcapi_cipher_stream_init_dec;
+	kcapi_cipher_stream_update;
+	kcapi_cipher_stream_op;
+	kcapi_aead_getdata_input;
```

Comment:
please add kcapi_cipher_[enc|dec]_aes_[cbc|ctr]

Reply:
oops, thanks for catching it